src/control_interface.cpp

```diff
@@ -219,6 +221,10 @@ void setJointMotorPower(robot::types::jointid_t joint, double power) {
 			Globals::wristKinematics.jointPowerToGearPower(jointPwr);
 		setMotorPower(motorid_t::wristDiffLeft, gearPwr.left);
 		setMotorPower(motorid_t::wristDiffRight, gearPwr.right);
+	} else if (joint == jointid_t::handActuator) {
+		if (std::abs(power) > 0.1) {
+			setMotorPos(motorid_t::handActuator, power > 0 ? 1 : 0);
+		}
```

Comment:
This code just sets the hand linear actuator in or out. The firmware of the actuator board (id 0x16) is configured to accept position 0 as retracted and anything else as extended.